test/steps/image_registry.go

```diff
@@ -24,26 +24,46 @@ import (
 	"github.com/cucumber/godog"
 	"github.com/kiegroup/kogito-cloud-operator/test/config"
 	"github.com/kiegroup/kogito-cloud-operator/test/framework"
+	"github.com/kiegroup/kogito-cloud-operator/test/steps/mappers"
 )
 
+/*
+	DataTable for Maven:
+	| -Dproperty=value |
+*/
+
 // registerImageRegistrySteps register all existing image registry steps
 func registerImageRegistrySteps(ctx *godog.ScenarioContext, data *Data) {
 	ctx.Step(`^Local example service "([^"]*)" is built by Maven using profile "([^"]*)" and deployed to runtime registry$`, data.localServiceBuiltByMavenWithProfileAndDeployedToRuntimeRegistry)
+	ctx.Step(`^Local example service "([^"]*)" is built by Maven using profile "([^"]*)" and deployed to runtime registry with options:$`, data.localServiceBuiltByMavenWithProfileAndDeployedToRuntimeRegistryWithOptions)
 }
 
 // Build local service and deploy it to registry if the registry doesn't contain such image already
 func (data *Data) localServiceBuiltByMavenWithProfileAndDeployedToRuntimeRegistry(contextDir, profile string) error {
+	return data.localServiceBuiltByMavenWithProfileAndDeployedToRuntimeRegistryWithOptions(contextDir, profile, nil)
+}
+
+// Build local service and deploy it to registry if the registry doesn't contain such image already
+func (data *Data) localServiceBuiltByMavenWithProfileAndDeployedToRuntimeRegistryWithOptions(contextDir, profile string, table *godog.Table) error {
+	var options []string
```

Comment:
here we should call it `mavenOptions` IMO

Reply:
Well, we don't call them mavenProfiles here it as well, so was keeping it consistent :) Is that OK? We have profiles and options and mavenProfiles and mavenOptions.

Reply:
oki good